src/main/java/pl/karolska/example/Car.java

```diff
@@ -12,10 +12,10 @@ public Car (){
     }
 
```

Comment:
Jeżeli to zadanie to miało być:
![image](https://user-images.githubusercontent.com/17989854/131245633-56f9279d-8ad3-446d-ac6d-5ebb21f88131.png)

to głowny problem tutaj jest to, że w commicie mamy więcej rzeczy niż tylko to konkretne zadanie.

![image](https://user-images.githubusercontent.com/17989854/131245648-e4d15191-be54-450e-823a-0bcb4e4aff7a.png)


Jak sobie klikniesz tutaj to zobaczysz jakie pliki są zmienione w tym pull requescie. Zobaczysz tam, że między innymi coś się dzieje z plikiem Car.java i tutaj pytanie - czemu cokolwiek w Car się zmienia w zadaniu, które w ogóle tego 'Car' nie dotyczy?